ru/mail/chrome/communicator/utilityOverlay.dtd

```diff
@@ -52,7 +52,7 @@
 <!ENTITY pac.label                                      "Онлайн (прокси: авто)">
 <!ENTITY pac.accesskey                                  "a">
 
-<!ENTITY proxy.label                                    "Конфигурация прокси">
+<!ENTITY proxy.label                                    "Конфигурация прокси…">
```

Comment:
"Proxy-server" sounds much more better, I agree, but taken into account UI space limitation the short form "Proxy" (in Russian language) is used even more, than a full term. Please see MS link above, I suggest to leave it as it is now.